website/docs/api/doc.md

```diff
@@ -751,22 +751,23 @@ The L2 norm of the document's vector representation.
+| `text`                               | A string representation of the document text. ~~str~~                                                                                           |
+| `text_with_ws`                       | An alias of `Doc.text`, provided for duck-type compatibility with `Span` and `Token`. ~~str~~                                                   |
+| `mem`                                | The document's local memory heap, for all C data it owns. ~~cymem.Pool~~                                                                        |
+| `vocab`                              | The store of lexical types. ~~Vocab~~                                                                                                           |
+| `tensor` <Tag variant="new">2</Tag>  | Container for dense vector representations. ~~numpy.ndarray~~                                                                                   |
+| `user_data`                          | A generic storage area, for user custom data. ~~Dict[str, Any]~~                                                                                |
+| `lang` <Tag variant="new">2.1</Tag>  | Language of the document's vocabulary. ~~int~~                                                                                                  |
+| `lang_` <Tag variant="new">2.1</Tag> | Language of the document's vocabulary. ~~str~~                                                                                                  |
+| `sentiment`                          | The document's positivity/negativity score, if available. ~~float~~                                                                             |
+| `user_hooks`                         | A dictionary that allows customization of the `Doc`'s properties. ~~Dict[str, Callable]~~                                                       |
+| `user_token_hooks`                   | A dictionary that allows customization of properties of `Token` children. ~~Dict[str, Callable]~~                                               |
+| `user_span_hooks`                    | A dictionary that allows customization of properties of `Span` children. ~~Dict[str, Callable]~~                                                |
+| `has_unknown_spaces`                 | Whether the document was constructed without known spacing between tokens (typically when created from gold tokenization). ~~bool~~             |
+| `_`                                  | User space for adding custom [attribute extensions](/usage/processing-pipelines#custom-components-attributes). ~~Underscore~~                   |
+| `activations`                        | A dictionary of activations per trainable pipe (available when the `store_activations` option of a pipe is enabled). ~~Dict[str, Option[Any]]~~ |
```

Comment:
add "new" tag for 4.0

Reply:
Also for the other updates to the docs :-)

Reply:
Added to all `activations` and `save_activations` attributes/options.